README.md

```diff
@@ -263,6 +263,10 @@ OPTIONS:
    --s3.bucket value The S3/minio bucket to use when using S3 proxy backend.
       [$BAZEL_REMOTE_S3_BUCKET]
 
+   --s3.bucket_lookup_type value The S3/minio bucket lookup type to use when
```

Comment:
github won't let me comment on lines that aren't close to the dirr, but could you also add bucket_lookup_type to the example s3_proxy yaml further down in this file?